vue/components/ui/generic-button-platforme/generic-button-platforme.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <button class="btn btn-content" v-bind:style="buttonStyleColor">
+        {{ buttonText }}
+    </button>
+</template>
+
+<style scoped>
+.btn-content:hover:not(:disabled)::after {
+    right: 4px;
+}
+
+.btn-content::after {
+    background-image: url(https://d3moonnr9fkxfg.cloudfront.net/royal-canin.sprite--brand3--xs.svg?v=8-7-8);
+    background-position: 45.45% 85.71%;
+    background-repeat: no-repeat;
+    content: "";
+    display: inline-block;
+    height: 32px;
+    overflow: hidden;
+    position: absolute;
+    right: -32px;
+    top: 50%;
+    -webkit-transform: translateY(-50%);
+    transform: translateY(-50%);
+    -webkit-transform-origin: 50% 50%;
+    transform-origin: 50% 50%;
+    -webkit-transition: 0.2s;
+    transition: 0.2s;
+    width: 32px;
+}
+
+.btn {
+    border: 2px solid transparent;
+    border-radius: 999px;
+    cursor: pointer;
+    display: inline-block;
+    font-size: 14px;
+    font-weight: 300;
+    line-height: inherit;
+    outline: none;
+    overflow: hidden;
+    padding: 6px 27px;
+    position: relative;
+    text-align: center;
+    text-decoration: none;
+    text-overflow: ellipsis;
+    -webkit-transition: 0.2s;
+    transition: 0.2s;
+    vertical-align: middle;
+    white-space: nowrap;
+    width: auto;
+}
+
+body {

```

Comment:
You can delete this, the font will be defined by the parent context. e.g.: each application defines its fonts in their own body